### DISCUSSION TITLE
The nextflow script reports an error in delivering the task on the slurm

### DISCUSSION THREAD

**Discussion by heweihuang** [nextflow-io/nextflow#2947] [Q&A]
The output of nextflow is in the root directory, and the task delivery qsub cannot be used. How to deal with these problems, thank you.  The content of the error is as follows:

Caused by:
  java.io.IOException: Cannot run program "qsub" (in directory "/test/91/f01bba18eb7a3075e4b09fbd921cba"): error=2, No such file or directory

Command executed:

  qsub -N nf-QC .command.run

**Comment by bentsherman** [comment]
Answering for posterity. The `qsub` command is the submit command for schedulers like PBS, but your title says you are using SLURM. I would make sure you are using the `slurm` executor in your Nextflow configuration.
